decidim-collaborations.gemspec

```diff
@@ -13,21 +13,14 @@ Gem::Specification.new do |s|
   s.required_ruby_version = '>= 2.3.1'
 
   s.name = 'decidim-collaborations'
-  s.summary = 'New feature that adds the possibility of having crowdfunding campaigns within participatory spaces'
-  s.description = s.summary
+  s.summary = "A decidim collaboration module"
+  s.description = 'Adds the possibility of having crowdfunding campaigns within participatory spaces'
 
-  s.files = Dir['{app,config,db,lib}/**/*', 'LICENSE-AGPLv3.txt', 'Rakefile', 'README.md']
+  s.files = Dir['{app,config,db,lib}/**/*', 'Rakefile', 'README.md']
```

Comment:
I don't know much about this. License shouldn't go in the files array?

Reply:
I'm about to open a PR in the `decidim-generators` repo to fix exactly this. You're absolutely right, I'll change it back.

Reply:
Ok, but I was just asking to learn 😆 I didn't know exactly what is the purpose of this array (I can figure out, but wouldn't be able to say if this file should be included or not)

Reply:
Ah, ok, sorry! :rofl: This array is the list of files that will be shipped with the gem. The LICENSE should always be shipped in my opinion.

Reply:
Opened a PR against decidim-generators to fix this! :)